notes.md

```diff
@@ -135,45 +135,52 @@ Figure: Block Header with 1 octet TrackNumber
  |       |V|   |U|
  +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 ```
-Figure: Block Header with 2 octets TrackNumber
+Figure: Block Header with 2-Octet TrackNumber
 
 where:
 
-{newline="true" spacing="normal"}
-Track Number: 8, 16, 24, 32, 40, 48 or 64 bits
-: an EBML VINT coded track number
+{newline="false" spacing="normal"}
+Track Number:
+: 8, 16, 24, 32, 40, 48, or 64 bits. An EBML VINT-coded track number.
```

Comment:
I'm slightly baffled why 56 bits is not included in the list 😂 Or rather, if it's a VINT, why 64 bits is there but 56 isn't — given as an 64 bit VINT sacrifices 8 bits to the length marker, leaving 56 bits for the value. Same applies to other places below.

Reply:
I plead guilty 88df108e245ccaa0024b5fe3dc66bfc1fc053d5d

Reply:
Fixed in #827